### DISCUSSION TITLE
helm chart deployment.

### DISCUSSION THREAD

**Discussion by venkatesh3889** [fluxcd/flux2#3916] [General]
Flux works with helm chart when helm chart uploads to OCI and from there flux finds the change and apply the changes to cluster. where as we have different requirement where we helm to deploy applications without updating them to OCI.

this is how we use helm:

1. We keep the helm values.yaml file in a repo and update the new image tag and deploy the same values.yaml file with help upgrade. 

Now we want to move the same process with gitops and flux2 cd. Can we use same approach of updating values.yaml file in repository and apply the changes to cluster with flux?

**Comment by stefanprodan** [comment]
> Can we use same approach of updating values.yaml file in repository and apply the changes to cluster with flux?

Yes, please see https://fluxcd.io/flux/guides/helmreleases/#refer-to-values-in-configmaps-generated-with-kustomize

**Comment by JayasreeReddy20** [comment]
Hi @stefanprodan, 
We have tried to replicate this in our tenant-configuration. Is it possible for flux to auto-update the **values.yaml** file inside the **chart** folder when we update our my-values.yaml with our required changes. Will this gonna work if we use this with below reference.
configMapGenerator:
  - name: podinfo-values
    files:
      - values.yaml=my-values.yaml
 How can my-values.yaml file changes gets reflected into the **./chart/values.yaml**  without running any helm updates?
Could you please give a brief info if you know how to proceed.
